UnitTests/UnitTests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>UnitTests Class Library</Description>
     <Authors>Administrator</Authors>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp3.1;net50</TargetFrameworks>
```

Comment:
`UnitTests` project is netcore only tests. 
.Net Framework tests living in 2 other projects, no need to include additional targets here.

Reply:
I completely missed the other SLNs and CSprojects, so I have removed the .NET 4.6.1 reference here.
I was unable to find any other project referencing .NET 5. Was there supposed to be one, or is it okay to leave the `net50` here?

Reply:
Why we may need to build unittests for .net5.0? 

There is nothing in UnitTests that specific to net5 at the moment (no conditional compilation statements). When you run against .net50 environment it will just fine run .net3.1 build.

Reply:
Currently there are indeed no conditional compiles for .NET 5 in either the unittest project or main library, otherwise this platform would have already been there as a target.
The reason you need it here in the target frameworks is so you can actually run the unittests from the .net5 runtime, using `dotnet test jose-jwt.sln --framework=net50` in the pipeline.
See https://github.com/dvsekhvalnov/jose-jwt/pull/168/files#diff-063ca77e012f959eba648db60e6868875fd1e70e5f5ac081193d848f8d61ef32R56

Reply:
ok, if we need it, than fine.